src/app/search/searchActions.js

```diff
@@ -61,43 +59,88 @@ const loadSearchResults = debounce((searchTerm, page, state, dispatch) => {
       type: SET_SEARCH_RESULTS,
       payload: {
         entries: [],
-        pageCount: 0,
       },
     })
     return
   }
 
-  const tilesets = searchableAndVisibleLayers.map((l) => l.tilesetId).join(',')
-  const baseURL = `${
-    process.env.REACT_APP_V2_API_ENDPOINT
-  }/tilesets/{{tilesets}}/search/?query={{query}}&limit={{limit}}&offset={{offset}}`
-  const url = buildEndpoint(baseURL, {
-    tilesets,
-    query: encodeURIComponent(searchTerm),
-    limit: SEARCH_MODAL_PAGE_SIZE,
-    offset: page * SEARCH_MODAL_PAGE_SIZE,
+  // NOW - error 500
+  // https://api-dot-skytruth-pleuston.appspot.com/v2/tilesets/gfw-tasks-657-uvi-v2,gfw-tasks-658-reefer-uvi-v4/search/?query=orca&limit=14&offset=0
```

Comment:
🤕

Reply:
Yeah this is meant to be removed - the whole multiple endpoints thing won't probably be necessary anyways.